ert3/engine/_run.py

```diff
@@ -167,7 +173,7 @@ def run(
         .set_type("function" if isinstance(stage, ert3.config.Function) else "unix")
     )
 
-    inputs = ert3.config.link_inputs(ensemble_config, stage)
+    inputs = ert3.config.link_inputs(experiment_run_config.ensemble_config, stage)
```

Comment:
Should this be a part of validation in this new configuration object?

The linking was meant to abstract away the fact that inputs are composites of multiple configurations, which is one purpose of the run configuration.

Linked inputs (or just inputs) can then be fetched from the run configuration after linking.

Reply:
That sounds like a good idea, I will look into that.

Reply:
Done.